experimental/PropertyDDS/packages/property-changeset/src/path_helper.js

```diff
@@ -332,6 +332,33 @@ PathHelper.quotePathSegment = function(in_pathSegment) {
   return '"' + in_pathSegment + '"';
 };
 
+/**
+ * Reverse a quoted/escaped string for a path seqment
+ *
+ * @param {string} in_quotedPathSegment   - The quoted/escaped path string to put in quotes
+ *
+ * @return {string} unquoted path string
+ */
+ PathHelper.unquotePathSegment = function(in_quotedPathSegment) {
+    if (!in_quotedPathSegment || typeof in_quotedPathSegment !== 'string') {
+        return in_quotedPathSegment;
+    }
+
+    // We remove double quotes: ;
+    if (in_quotedPathSegment[0] === '"' && in_quotedPathSegment[in_quotedPathSegment.length - 1] === '"') {
+        in_quotedPathSegment = in_quotedPathSegment.substr(1, in_quotedPathSegment.length - 2);
+    }
+
+    // Then we unescape escape symbols
```

Comment:
Strictly speaking, this should only happen when the path actually is quoted. I think it will always work, since you should never have an unqouted path with those characters in it, still it would be better to move this into the if block above.